pkg/cmd/roachtest/tests/acceptance.go

```diff
@@ -68,6 +68,14 @@ func registerAcceptance(r registry.Registry) {
 				timeout: 30 * time.Minute,
 			},
 		},
+		registry.OwnerDisasterRecovery: {
+			{
+				name:     "c2c",
+				fn:       runAcceptanceClusterReplication,
+				numNodes: 3,
```

Comment:
Do we need 3 nodes? Can we run the workload on the src node? I suppose since this is "local" it doesn't really matter?